nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/zookeeper.properties

```diff
@@ -27,6 +27,28 @@ tickTime=2000
 dataDir=./state/zookeeper
 autopurge.snapRetainCount=30
 
+# Embedded/distributed ZK TLS connection support can be activated by setting these properties at minimum:
+#
+# secureClientPort=2281
+# serverCnxnFactory=org.apache.zookeeper.server.NettyServerCnxnFactory
+
+# Most TLS configurations will set these values as well:
```

Comment:
Good call on including the example properties. We should include a note about not using the reconfig-style server.N entries if both clientPort and secureClientPort are being used.

Reply:
Agreed.